test/blocklib/core/selector.hpp

```diff
@@ -0,0 +1,158 @@
+#ifndef GRAPH_PROTOTYPE_SELECTOR_HPP
+#define GRAPH_PROTOTYPE_SELECTOR_HPP
+
+#include <node.hpp>
+#include <utils.hpp>
+
+namespace gr::blocks::basic {
+using namespace fair::graph;
+
+// optional shortening
+template<typename T, fair::meta::fixed_string description = "", typename... Arguments>
+using A           = Annotated<T, description, Arguments...>;
+
+using SelectorDoc = Doc<R""(
+@brief basic multiplexing class to route arbitrary inputs to outputs
+)"">;
+
+template<typename T>
+struct Selector : node<Selector<T>, SelectorDoc> {
```

Comment:
Please add the documentation of and reference to https://wiki.gnuradio.org/index.php/Selector here.

Since we cannot (yet) paste images here, and updated description text and examples for the (in,out) tuple pairs would be helpful.  N.B. just realised that the GR 3.X block doesn't seem to support (at least it's not documented) the optional 'backPressure' behaviour. We may have added some new functionality.